README.md

```diff
@@ -83,6 +83,35 @@ message.setRoomId(room.getId());
 message.setFiles(URI.create("http://example.com/hello_world.jpg"));
 spark.messages().post(message);
 
+// Share an adaptive card with the room
+message = new Message();
+message.setRoomId(room.getId());
+message.setText("Mandatory fallback text");
+JsonArray attachmentsArry = Json.createArrayBuilder()
+                .add(Json.createObjectBuilder()
+                        .add("contentType", "application/vnd.microsoft.card.adaptive")
+                        .add("content", Json.createObjectBuilder()
+                                .add("$schema", "http://adaptivecards.io/schemas/adaptive-card.json")
+                                .add("type", "AdaptiveCard")
+                                .add("version", "1.0")
+                                .add("body", Json.createArrayBuilder()
```

Comment:
The code looks great! We'd like to request some additional details in regards to the methods that actually handle creating the cards. Below is an example:

```java
.add("body", Json.createArrayBuilder() // create initial card object
    .add(Json.createObjectBuilder() // create a card child object to store the text
    .add("type", "TextBlock")
    .add("text", "Here is a ninja cat")
    .build() // build the card child object
)
.add(Json.createObjectBuilder() // create a card child object to store the image
    .add("type", "Image")
    .add("url", "http://adaptivecards.io/content/cats/1.png")
    .build() // build the card child object
)
.build() // build the card object
```

Feel free to change any details in the example as needed. Providing clarity on the important sections of the example would help other developers identify the portion of code that is used to specifically generate the cards.

Reply:
@InteractiveTimmy I've added relevant comments which should clarify how the JSON array is built. Please take a look and let me know if there's anything else.